Source/Scene/processPbrMaterials.js

```diff
@@ -600,19 +616,13 @@ define([
 
             // Generate fragment shader's lighting block
             // The Sun is brighter than your average light source, and has a yellowish tint balanced by the Earth's ambient blue.
+            fragmentShader += '#ifndef USE_CUSTOM_LIGHT_COLOR \n';
             fragmentShader += '    vec3 lightColor = vec3(1.5, 1.4, 1.2);\n';
+            fragmentShader += '#else \n';
```

Comment:
The default light color could also share the `gltf_lightColor` uniform.

Then `USE_CUSTOM_LIGHT_COLOR` might be just `USE_LIGHT_COLOR`, and switched off the user sets the color to (0,0,0).

Reply:
Looking at https://github.com/AnalyticalGraphicsInc/cesium/pull/7017/files#r216167338 and the Color doc, is this because colors are not allowed to have components greater than 1?

Reply:
I'm not sure about that comment. Can you explain @ggetz? Colors aren't supposed to have values >1 but you can as long as you aren't converting the components to bytes. Perhaps this should be a `Cartesian3`?

Reply:
I'd also like to keep the uniform the way it is since the compiler could optimize out the uniform when not set.

Reply:
It was because I was using a Color with components greater than 1. `Cartesian3` would work. :+1: